Configure

```diff
@@ -501,6 +502,9 @@ my @disable_cascades = (
 
     "crypto-mdebug"     => [ "crypto-mdebug-backtrace" ],
 
+    # scrypt requires PBKDF2
+    "pbkdf2"            => [ "scrypt" ],
```

Comment:
I wondered too. Why would we need to disable this? It complicates matters and adds additional needs for #ifdefs for it. We didn't have an option to disable it in 1.1.0